src/router/app.tsx

```diff
@@ -1,15 +1,16 @@
 import { Fragment, useContext } from "react";
 import { Navigate, Route, Routes } from "react-router-dom";
 import { AuthContext } from "../contexts/auth.context";
-import { AvatarCreatePage } from "../pages/avatar-create.page";
-import { AvatarResultsPage } from "../pages/avatar-results.page";
 import { LoadingPage } from "../pages/loading.page";
+import { LoginPage } from "../pages/login.page";
 import { PersonalPage } from "../pages/personal.page";
-import { SignupPage } from "../pages/signup.page";
+import { ProfileCreatePage } from "../pages/profile-create.page";
+import { ProfileResultsPage } from "../pages/profile-results.page";
 import { TeamPage } from "../pages/team.page";
 
 export const App = () => {
   const { isAuthenticated, user, isLoadingContext } = useContext(AuthContext);
+  const hasProfile = user.avatar && user.role;
```

Comment:
Will this cause that error where user is undefined? Ignore this if it's not an issue anymore or in later PRs